examples/parquet_write_parallel/src/main.rs

```diff
@@ -134,7 +133,14 @@ fn create_batch(size: usize) -> Result<Chunk> {
         })
         .collect();
 
+<<<<<<< HEAD
```

Comment:
It looks like merge conflicts were still in the commit. @jorgecarleitao

Reply:
![6jiwtn](https://user-images.githubusercontent.com/2772607/173385004-f7082743-407d-4c55-aa88-e0e24412f17f.jpg)